src/commands/run.rs

```diff
@@ -330,6 +335,16 @@ async fn build_store_instance(
                     linked_capabilities.insert("http-client".to_string());
                 }
             }
+            #[cfg(feature = "gpio")]
```

Comment:
Could you please add at least one intergration test to this new capability so that it will be automatically tested in our CI/CD pipelines?